docs/setup-and-installing.md

```diff
@@ -1,153 +1,81 @@
-
-1. Build an assembly for the deployment
-<br>The first step of running a cloud deployment is uploading all the files that your game uses. This includes executable files for the clients and workers, and the assets your workers use (like models and textures used by the Unity client to visualise the game). We call that set of game files an assembly.
-<br><br> To build an assembly for your game to use while running in the cloud, either:
-    - In the Unity Editor, select **SpatialOS** > **Build all workers for cloud**
-
-1. Upload the assembly to the cloud
-
-Open a terminal and navigate to the `UnityGDK` directory (the repository you’ve cloned).
-    Run `spatial cloud upload <assembly name>`.
-    > The `<assembly name>` is just a label so you can identify this assembly in the next step - for example, `MyGDKAssembly`.
-
-> **It’s done when:** You see `spatial upload MyGDKAssembly succeeded` printed in your console output.
-
-1. Launch a cloud deployment
-<br>The command `spatial cloud launch` deploys a project to the cloud. Its full syntax is:
-
-    `spatial cloud launch <assembly name> <launch configuration> <deployment name> --snapshot=<snapshot file>`
-
-    where:
-
-    - `<assembly name>` is the name of the assembly the deployment will use (the one you named in step 3).
-    - `<launch configuration>` is the configuration file for the deployment. This project includes one called `default_launch.json`.
-    - `<deployment name>` is a name of your choice, which you’ll use to identify the deployment. This must be in lowercase.
-    - `<snapshot file>` is the snapshot of the world you want to start from. This project includes one called `snapshots/default.snapshot`.
-
-    In the same terminal window, run `spatial cloud launch <assembly name> cloud_launch.json <deployment name> --snapshot=snapshots/default.snapshot`
-
-    This command defaults to deploying to clusters located in the US. So if you’re in Europe, add the `--cluster_region=eu` flag for better performance.
-
-> **It’s done when:** You see `Successfully created deployment` printed in your console output.
-
-
-1. Launch a game client:
-    1. Open https://console.improbable.io/projects.
-    <br>You’ll see the project and the deployment you just created.
-
-    1. Click the deployment’s name to open the overview page.
-
-    1. To get links to share the game with others, click **Share**.
-
-    1. Click **Launch**.
-        > You can ignore the prompt to install the Launcher, as it’s installed as part of SpatialOS.
-
-    1. Once you’ve finished playing, click **Stop** in the Console.
+### Sign up for a SpatialOS account
+
+You'll need a SpatialOS account in order to run the SpatialOS GDK for Unity.
+Follow [this link](https://improbable.io/get-spatialos) to sign up for free.
+
+### Set up your machine
+
+#### Windows
+
+1. [Unity 2018.2.8](https://unity3d.com/get-unity/download/archive) 
+    - Make sure to select the following components:
```

Comment:
...components during installation.

(Makes it clear where you get the components.)